com.unity.render-pipelines.high-definition/Editor/Sky/HDRISky/HDRISkyEditor.cs

```diff
@@ -81,7 +81,7 @@ public override void OnEnable()
             var hdrp = HDRenderPipeline.defaultAsset;
             if (hdrp != null)
                 m_IntegrateHDRISkyMaterial = CoreUtils.CreateEngineMaterial(hdrp.renderPipelineResources.shaders.integrateHdriSkyPS);
-            m_ReadBackTexture = new Texture2D(1, 1, TextureFormat.RGBAFloat, false, false);
+            m_ReadBackTexture = new Texture2D(1, 1, GraphicsFormat.R32G32B32A32_SFloat, TextureCreationFlags.None);
```

Comment:
Before this was not linear, now seems it is, again as before do we have an option?